admin/admin-domain-service/ballerina/resources/admin-api.yaml

```diff
@@ -2719,6 +2719,193 @@ paths:
           -F "type=WSO2-IS" "https://127.0.0.1:9443/api/am/admin/v3/key-managers/discover"'
       operationId: getWellKnownInfoKeyManager
 
+######################################################
+  # The "Organization" resource API
+######################################################
+  /organization:
```

Comment:
this might need to be organizations, plural

Reply:
Fix https://github.com/wso2/apk/pull/630/commits/eaca1b60862435210c947e860d55bb6d711a23aa